argparse/argparse.py

```diff
@@ -3,7 +3,10 @@
 """
 
 import sys
-from _collections import namedtuple
+try:
+    from _collections import namedtuple
+except ImportError:
+    from collections import namedtuple
```

Comment:
Fairly speaking, I have a similar change for gzip.py - to make it easier to test it with CPython (i.e. against CPython behavior). The fact that I don't merge it for months says that I'm not really sure if it's good or bad...